extide/gradle/src/org/netbeans/modules/gradle/loaders/LegacyProjectLoader.java

```diff
@@ -194,7 +199,15 @@ private static GradleProject loadGradleProject(ReloadContext ctx, CancellationTo
         }
         return ret;
     }
-    
+
+    private static GradleVersion compatibleGradleVersion(Project prj) {
```

Comment:
Nitpick: Does this return a gradle version **compatible** with the current project, or the **configured** one ? I mean if the project does not specify anything, the current default version is returned regardless of project's structure

Reply:
Oh, good catch, I left the checking of the IDE Java Version out from this computation!